CHANGES.md

```diff
@@ -53,8 +53,8 @@
 
 ## Highlights
 
-* New highly anticipated feature X added to Python SDK ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
-* New highly anticipated feature Y added to Java SDK ([BEAM-Y](https://issues.apache.org/jira/browse/BEAM-Y)).
+* Initial preview of the [RunInference](https://s.apache.org/inference-sklearn-pytorch) API added to Python SDK ([BEAM-13970](https://issues.apache.org/jira/browse/BEAM-13970)).
```

Comment:
Do we want to call it a preview here?

Reply:
Bad wording on my part. How does this sound: "Added the new RunInference API to the Python SDK. Users can now make inferences with PyTorch and Scikit-learn models!". Open to suggestions/changes.

Reply:
Yeah that sounds better to me. You might also check with @rezarokni to see how he wants to message it. (might need to reach out to him directly, I don't know that he'll see a ping from GitHub).

Reply:
How about "Added [RunInference](https://s.apache.org/inference-sklearn-pytorch) API, a framework agnostic transform for inference. With this release PyTorch(<config link>)  and Scikit-learn (<config link>) are supported by the transform.